funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_manager.py

```diff
@@ -206,6 +206,10 @@ def __init__(self,
             args=(self._kill_event,)
         )
 
+        self.poller = zmq.Poller()
+        self.poller.register(self.task_incoming, zmq.POLLIN)
+        self.poller.register(self.funcx_task_socket, zmq.POLLIN)
```

Comment:
I am not too sure here. Do we need to move this out of the pull_tasks?

Reply:
I think this change is reasonable.